Documentation/sources/content-card-ui/tutorial/displaying-content-cards.md

```diff
@@ -0,0 +1,139 @@
+            }
+        }
+    }
+
+    // Function to refresh the aepUIList from the ContentCardUIProvider
+    fun refreshContent() {
+        viewModelScope.launch {
+            contentCardUIProvider.refreshContent()
+        }
+    }
+}
+```
+
+> Note - only content cards for which the user has qualified are returned by the getContentCardUI API. Client-side rules are defined in the Adobe Journey Optimizer campaign.
+
+## Display Content Cards
+
+To display content cards in your app, use the `AepUI` objects returned by the `getContentCardsUI` API. The `AepUI` objects provide the user interface for templated content cards in your application.
+
+### Display Content Cards in Compose UI application
+
+Below is an example of how to display content cards in a Compose UI application:
+
+```kotlin
+override fun onCreate(savedInstanceState: Bundle?) {
+  super.onCreate(savedInstanceState)
+
+  binding = ActivityScrollingBinding.inflate(layoutInflater)
+  setContentView(binding.root)
+
+  val surfaces = mutableListOf<Surface>()
+  val surface = Surface("homepage")
+  surfaces.add(surface)
+
+  // Initialize the ContentCardUIProvider
+  contentCardUIProvider = ContentCardUIProvider(surface)
+
+  // Initialize the ViewModel
+  contentCardViewModel =
+  ViewModelProvider(this, AepContentCardViewModelFactory(contentCardUIProvider)).get(
+    AepContentCardViewModel::class.java
+  )
+
+  contentCardCallback = ContentCardCallback()
+
+  // Set a click listener for refresh button which calls the API for fetch content cards from Edge
+  val refreshButton: ImageButton = findViewById(R.id.refreshButton)
+  refreshButton.setOnClickListener {
+    Messaging.updatePropositionsForSurfaces(surfaces)
+    contentCardViewModel.refreshContent()
+  }
+
+  binding.composeView.apply {
+    setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+    setContent {
+      AppTheme {
+        AepContentCardList(contentCardViewModel)
+      }
+    }
+  }
+}
+
+@Composable
+private fun AepContentCardList(viewModel: AepContentCardViewModel) {
+  // Collect the state from ViewModel
+  val aepUiList by viewModel.aepUIList.collectAsStateWithLifecycle()
+  
+  // Create row with composables from AepUI instances
+  LazyRow {
+    items(aepUiList) { aepUI ->
+                               when (aepUI) {
+                                 is SmallImageUI -> {
+                                   val state = aepUI.getState()
+                                   if (!state.dismissed) {
+                                     SmallImageCard(
+                                       ui = aepUI,
+                                       style = smallImageCardStyleRow,
+                                       observer = ContentCardEventObserver(contentCardCallback)
+                                     )
+                                   }
+                                 }
+                               }
+                              }
+  }
+}    
```

Comment:
Can we keep only this much from the code snippet and remove the rest? Not all apps might call it in Activity's `onCreate`. It should be sufficient to show how to use the composables in a list
https://wiki.corp.adobe.com/display/adms/Using+Content+Cards+with+UI+on+Android#UsingContentCardswithUIonAndroid-DisplayContentCards